### ISSUE TITLE
Consider wider distribution

### ISSUE THREAD

**Issue by adamtabrams** [akupila/gitprompt#2]
Would you consider getting this added to some official Linux repos? Or possibly to the Arch User Repo (the AUR likely takes less effort)?

**Comment by akupila** [comment]
Yes definitely! I'm not sure what the process is there though; would be happy to accept a PR

**Comment by akupila** [comment]
Related: https://github.com/goreleaser/goreleaser/issues/641
